_topic_maps/_topic_map.yml

```
@@ -21,6 +21,8 @@ Topics:
   File: ossm-about-openshift-service-mesh
 - Name: Understanding Service Mesh
   File: ossm-about-concepts-assembly
+- Name: Supported Platforms and Configurations
```

See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings:
```suggestion
- Name: Supported platforms and configurations
```